arquillian-tests fails to resolve net.wasdev.wlp.starters.rest:provided-pom:0.0.3

Since app-accelerator is shutdown at the end of March 2018. Dependencies in the arquillian-tests can not be resolved from this repository
```
       <repositories>
                <repository>
                        <id>liberty-starter-maven-repo</id>
                        <name>liberty-starter-maven-repo</name>
                        <url>http://liberty-starter.wasdev.developer.ibm.com/start/api/v1/repo</url>
                </repository>
        </repositories>
```

Error in the build log:

[INFO] [ERROR] Failed to execute goal on project test-configure-arquillian: Could not resolve dependencies for project net.wasdev.wlp.maven.it:test-configure-arquillian:liberty-assembly:1.0.0.Final: The following artifacts could not be resolved: net.wasdev.wlp.starters.rest:provided-pom:pom:0.0.3, net.wasdev.wlp.starters.persistence:provided-pom:pom:0.0.3, net.wasdev.wlp.starters.rest:runtime-pom:pom:0.0.3, net.wasdev.wlp.starters.persistence:runtime-pom:pom:0.0.3: Failure to find net.wasdev.wlp.starters.rest:provided-pom:pom:0.0.3 in http://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]

--- a/liberty-archetype-mp/pom.xml
+++ b/liberty-archetype-mp/pom.xml
@@ -5,7 +5,7 @@
     <parent>
         <groupId>net.wasdev.wlp.maven</groupId>
         <artifactId>liberty-maven</artifactId>
-        <version>2.1.3-SNAPSHOT</version>
+        <version>2.3-SNAPSHOT</version>
     </parent>
 
     <artifactId>liberty-archetype-mp</artifactId>

--- a/liberty-maven-plugin/src/it/arquillian-tests/pom.xml
+++ b/liberty-maven-plugin/src/it/arquillian-tests/pom.xml
@@ -121,21 +121,25 @@
 			<groupId>javax.ws.rs</groupId>
 			<artifactId>javax.ws.rs-api</artifactId>
 			<version>2.0.1</version>
+			<scope>provided</scope>
 		</dependency>
 		<dependency>
 			<groupId>javax.json</groupId>
 			<artifactId>javax.json-api</artifactId>
 			<version>1.0</version>
+			<scope>provided</scope>
 		</dependency>
 		<dependency>
 			<groupId>com.ibm.websphere.appserver.api</groupId>
 			<artifactId>com.ibm.websphere.appserver.api.jaxrs20</artifactId>
 			<version>1.0.10</version>
+			<scope>provided</scope>
 		</dependency>
 		<dependency>
 			<groupId>com.ibm.websphere.appserver.api</groupId>
 			<artifactId>com.ibm.websphere.appserver.api.json</artifactId>
 			<version>1.0.10</version>
+			<scope>provided</scope>
 		</dependency>
 
 		<!-- Persistence dependencies -->
@@ -143,11 +147,13 @@
 			<groupId>com.ibm.websphere.appserver.api</groupId>
 			<artifactId>com.ibm.websphere.appserver.api.persistence</artifactId>
 			<version>1.0.10</version>
+			<scope>provided</scope>
 		</dependency>
 		<dependency>
 			<groupId>org.eclipse.persistence</groupId>
 			<artifactId>javax.persistence</artifactId>
 			<version>2.1.0</version>
+			<scope>provided</scope>
 		</dependency>
 
 	</dependencies>

--- a/liberty-maven-plugin/src/it/arquillian-tests/pom.xml
+++ b/liberty-maven-plugin/src/it/arquillian-tests/pom.xml
@@ -110,39 +110,46 @@
 			<scope>provided</scope>
 		</dependency>
 		<dependency>
-			<groupId>net.wasdev.wlp.starters.rest</groupId>
-			<artifactId>provided-pom</artifactId>
-			<version>0.0.3</version>
-			<type>pom</type>
-			<scope>provided</scope>
-		</dependency>
-		<dependency>
-			<groupId>net.wasdev.wlp.starters.persistence</groupId>
-			<artifactId>provided-pom</artifactId>
-			<version>0.0.3</version>
-			<type>pom</type>
-			<scope>provided</scope>
-		</dependency>
-		<dependency>
-			<groupId>net.wasdev.wlp.starters.rest</groupId>
-			<artifactId>runtime-pom</artifactId>
-			<version>0.0.3</version>
-			<type>pom</type>
-			<scope>runtime</scope>
-		</dependency>
-		<dependency>
-			<groupId>net.wasdev.wlp.starters.persistence</groupId>
-			<artifactId>runtime-pom</artifactId>
-			<version>0.0.3</version>
-			<type>pom</type>
-			<scope>runtime</scope>
-		</dependency>
-		<dependency>
 			<groupId>net.wasdev.wlp.common</groupId>
 			<artifactId>ci.common</artifactId>
 			<version>1.0</version>
 			<scope>test</scope>
 		</dependency>
+
+		<!-- Rest dependencies -->
+		<dependency>
+			<groupId>javax.ws.rs</groupId>
+			<artifactId>javax.ws.rs-api</artifactId>
+			<version>2.0.1</version>
+		</dependency>
+		<dependency>
+			<groupId>javax.json</groupId>
+			<artifactId>javax.json-api</artifactId>
+			<version>1.0</version>
+		</dependency>
+		<dependency>
+			<groupId>com.ibm.websphere.appserver.api</groupId>
+			<artifactId>com.ibm.websphere.appserver.api.jaxrs20</artifactId>
+			<version>1.0.10</version>
+		</dependency>
+		<dependency>
+			<groupId>com.ibm.websphere.appserver.api</groupId>
+			<artifactId>com.ibm.websphere.appserver.api.json</artifactId>
+			<version>1.0.10</version>
+		</dependency>
+
+		<!-- Persistence dependencies -->
+		<dependency>
+			<groupId>com.ibm.websphere.appserver.api</groupId>
+			<artifactId>com.ibm.websphere.appserver.api.persistence</artifactId>
+			<version>1.0.10</version>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.persistence</groupId>
+			<artifactId>javax.persistence</artifactId>
+			<version>2.1.0</version>
+		</dependency>
+
 	</dependencies>
 	<build>
 		<plugins>

--- a/liberty-maven-plugin/src/it/arquillian-tests/src/main/liberty/config/server.xml
+++ b/liberty-maven-plugin/src/it/arquillian-tests/src/main/liberty/config/server.xml
@@ -20,6 +20,7 @@
 		<feature>localConnector-1.0</feature>
 		<feature>jpa-2.1</feature>
 		<feature>jaxrs-2.0</feature>
+		<feature>jsonp-1.0</feature>
 		<feature>ejbLite-3.2</feature>
 	</featureManager>
 	

